### PR TITLE
UI Changes

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@akatsoulas r?

**Comment by johngian** [comment]
Let's stick to `Open Sans` and use it everywhere. `Contribute` and `About` titles have different `font-family`.

File: feedthefox/base/templates/jinja2/base.html [hunk 1]
```diff
@@ -38,7 +38,19 @@
...
+                    Resources
+                  </a>
+                  <ul class="dropdown-menu">
+                    <li><a href="{{ url('porting') }}">Porting Guide</a></li>
+                    <li><a href="{{ url('gaia') }}">Gaia Development</a></li>
+                    <li><a href="{{ url('addons') }}">Add-ons</a></li>
+                    <li><a href="{{ url('b2gdroid') }}">B2GDroid</a></li>
+                  </ul>
+                </li>
+                <li class="hidden-sm"><a href="{{ url('dashboard') }}">Dashboard</a></li>
```
Comment (johngian) [inline feedthefox/base/templates/jinja2/base.html:53]: ~~In mobile view this is not hidden.~~

Comment (comzeradd) [inline feedthefox/base/templates/jinja2/base.html:53]: That's expected behavior. We only hide this on tablet resolutions where tabzilla minimizes available navbar space.

**Comment by comzeradd** [comment]
> Let's stick to Open Sans and use it everywhere. Contribute and About titles have different font-family.

Added a commit to override bootstrap's font definitions for headings.

**Comment by johngian** [comment]
r+wc

**Pull request merged**